react-frontend/src/components/DigitalFramework/digitalframework.js

```diff
@@ -100,7 +103,7 @@ const DigitalFramework = () => {
           'This is the Digital Framework: our plan for becoming a digital government. We consider it a draft – an “alpha” version – that reserves the flexibility to learn and adapt as we go.'
         }
       />
-      <div className="digitalBody">
+      <SimplePageBodyDigital>
```

Comment:
Same comment here as [here](https://github.com/bcgov/digital.gov.bc.ca/pull/363#discussion_r504142927) -- is this a good opportunity to reduce everything to a single `SimplePageBody` object?

Reply:
It is for a future PR, this imports the existing styling without changing it. Refactoring every page would be a massive PR since the styling of the new SimplePageBody will probably introduce bugs in the the page.